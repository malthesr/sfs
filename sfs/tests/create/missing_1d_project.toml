args = ["create", "-vv", "--project", "7", "tests/create/missing.bcf"]
