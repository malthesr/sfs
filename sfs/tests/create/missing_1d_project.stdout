#SHAPE=<7>
1.000000 0.107143 1.535714 0.714286 1.071429 0.535714 0.035714
